Keda.Cosmosdb.Scaler/src/Protos/externalscaler.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+option csharp_namespace = "Keda.Cosmosdb.Scaler.Protos";
+
+package externalscaler;
+option go_package = ".;externalscaler";
+
+service ExternalScaler {
+    rpc IsActive(ScaledObjectRef) returns (IsActiveResponse) {}
+    rpc StreamIsActive(ScaledObjectRef) returns (stream IsActiveResponse) {}
+    rpc GetMetricSpec(ScaledObjectRef) returns (GetMetricSpecResponse) {}
+    rpc GetMetrics(GetMetricsRequest) returns (GetMetricsResponse) {}
+}
+
+message ScaledObjectRef {
+    string name = 1;
+    string namespace = 2;
+    map<string, string> scalerMetadata = 3;
```

Comment:
nit: Following the  [style guide](scalerMetadata)rename fields. For example: `scaler_metadata`

Reply:
Updating proto file now would be a breaking change. Is that possible? If not we should stick to existing naming

Reply:
Right, will have to see how much of a breaking change it is. We can definitely change it whenever we do v3, but for this PR we should keep it the same for now.